gulpfile.js

```diff
@@ -0,0 +1,42 @@
+var gulp = require('gulp');
+
+var rollup = require('rollup-vinyl-stream');
+var watch = require('gulp-watch');
+
+let cache;
+
+const _ru_base = function(rollup_config) {
+  return rollup({
+      config: 'rollup/' + rollup_config,
+      cache,
+    })
+    .on('bundle', (bundle) => cache = bundle)
+    .pipe(gulp.dest('./'));
+};
+
+gulp.task('ru_automation', function() {
+  return _ru_base('automation.js');
+});
+
+gulp.task('ru_core', function() {
+  return _ru_base('core.js');
+});
+
+gulp.task('ru_compatibility', function() {
+  return _ru_base('compatibility.js');
+});
+
+gulp.task('ru_demo', function() {
+  return _ru_base('demo.js');
+});
+
+gulp.task('ru_all', [
```

Comment:
We should be able to make this easier. Rollup has introduced multiple config support. We just need 1 file to export an array of configs.